app/dashboard/sync/binance.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+from django.utils import timezone
+
+import requests
+from dashboard.sync.helpers import record_payout_activity
+
+logger = logging.getLogger(__name__)
+
+headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0. 2272.118 Safari/537.36.'}
```

Comment:
Curious -> do they block us if we don't have a useragent ?

Reply:
i don't think so. i didn't find any info on that. do we stick to `Host: gitcoin.co` ?

here's their gist on API usage

```
The Binance Smart Chain Developer APIs are provided as a community service and without warranty, so please use what you need and no more. We support both GET/POST requests and there is a rate limit of 5 calls per sec/IP.

Note: Source attribution via a link back or mention that your app is "Powered by BscScan.com APIs" is required except for personal/private usage.
```

Reply:
Yup let's stick to that cause that way if we run into trouble and get it resolved!

Reply:
@thelostone-mc

**update:** we reverted to browser header after the smart chain explorer kept bouncing requests with error 403.